src/components/BasketballCourt/CircleArea.tsx

```diff
@@ -9,7 +9,7 @@ interface CircleAreaProps {
 }
 
 const CircleArea: React.FC<CircleAreaProps> = ({ startPoint }) => {
-  const { courtAreaXLength, threePointLineToCourtEdgeLenth, threePointLineRadius, circleRadius } =
+  const { courtAreaXLength, threePointLineToCourtEdgeLength, threePointLineRadius, circleRadius } =
```

Comment:
Will be better to create a new ticket to fix the typos

Reply:
Sure. Will fix this tomorrow. Cheers.